### PR TITLE
Adding new header comments to all the source files.

### PR DESCRIPTION
**Pull request by mcmillan03**
Request merge into stable (and move the 1.0 tag) and also merge to dev branch

### CODE REVIEW

File: src/benchmark/bc_demo.c [hunk 1]
```diff
@@ -2,10 +2,14 @@
 // LAGraph/src/demo/bc_demo: betweenness centrality for the GAP benchmark
```
Comment (jim22k) [inline src/benchmark/bc_demo.c]: LAGraph/src/benchmark/bc_demo

File: src/benchmark/mtx2bin_demo.c [hunk 2]
```diff
@@ -2,10 +2,14 @@
 // mtx2bin: convert Matrix Market file to SuiteSparse:GraphBLAS binary file
```
Comment (jim22k) [inline src/benchmark/mtx2bin_demo.c]: LAGraph/src/benchmark/mtx2bin_demo.c

File: src/benchmark/ss2_demo.c [hunk 3]
```diff
@@ -2,10 +2,14 @@
 // test_sssp: test for LAGraph
```
Comment (jim22k) [inline src/benchmark/ss2_demo.c]: LAGraph/src/benchmark/ss2_demo.c

**Review by jim22k — APPROVED** [review]
Found a couple of name mismatches. Not sure how much we care about exactly matching the filename. There are likely others that I missed. I wasn't looking too closely as I scanned the changes.
Overall looks good to me.

**Comment by mcmillan03** [comment]
Fixed the incorrect filenames noted by Jim.  This branch should be merged to dev and then released to stable.

**Review by DrTimothyAldenDavis — APPROVED** [review]
Looks fine to me.

**Pull request merged**